package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Learn how to use istanbul for tracking test/code coverage in your JS projects",
   "main": "test.js",
   "scripts": {
-    "coverage": "node ./node_modules/.bin/istanbul cover ./test.js",
+    "coverage": "istanbul cover ./test.js",
```

Comment:
@indieforger agree that this is simpler. 
the *reason* we used the full path was so people are not confused as to the location of the Istanbul script and don't end up installing it *globally*...